python/lambda.py

```diff
@@ -0,0 +1,5 @@
+def lambda_handler(event, context):
```

Comment:
you have lambda code but no lambda function in terraform?

Reply:
i wanted to make lambda a second branch.👍